packages/rocketchat-lib/server/methods/globalAnnouncement.js

```diff
@@ -0,0 +1,20 @@
+Meteor.methods({
+	globalAnnouncementRead() {
+		if (!Meteor.userId()) {
+			return false; //shall always be shown unless confirmed by the logged-in-user
+		}
+
+		const settingGlobalAnnouncement = RocketChat.models.Settings.findOne({_id: 'Layout_Global_Announcement'});
+		const user = RocketChat.models.Users.findOne({_id: Meteor.userId()});
+
+		return !!user.globalAnnouncementRead && (user.globalAnnouncementRead > settingGlobalAnnouncement._updatedAt);
```

Comment:
@geekgonecrazy comparing with `_updatedAt` has the flaw of making the annoucement re-appear on server startup.
If the settings were not `upserted` [on `add()`](https://github.com/assistify/Rocket.Chat/blob/core/%238446-global-announcement/packages/rocketchat-lib/server/functions/settings.js#L123), but only if the value set differs, this would not be the case anymore.

Reply:
I think this is ok for merging.  Maybe we should take a look at modding that behavior though..